packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/lib_auth/models/request_models/authenticate_user_request_model.dart

```diff
@@ -6,11 +6,17 @@ part 'authenticate_user_request_model.g.dart';
 
 @JsonSerializable(includeIfNull: false)
 class AuthUserRequestModel {
-  AuthUserRequestModel({this.username, this.password, this.refreshToken});
+  AuthUserRequestModel({
+    this.username,
+    this.password,
+    this.refreshToken,
+    this.token,
+  });
 
   final String? username;
   final String? password;
   final String? refreshToken;
+  final String? token;
```

Comment:
It looks like that you have mixed the authentication process with the one that retrieves a new access token. Please create separate models for those two flows

Reply:
removed